buildSrc/src/testKit/jdk-download/subproj/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'elasticsearch.jdk-download'
+}
+
+
+String fakeJdkVersion = Objects.requireNonNull(System.getProperty('tests.jdk_version'))
+jdks {
+    linux_jdk {
```

Comment:
nit: could we remove `_jdk` from the name to avoid redundancy ? And keep that as the naming format in general.

Reply:
This is just an example in a test, but I removed it.